ibis/impala/api.py

```diff
@@ -116,3 +117,42 @@ def connect(host='localhost', port=21050, database='default', timeout=45,
         raise
 
     return client
+
+
+def odbc_connect(dsn=None, connection_string=None, turbodbc_options=None,
+                 hdfs_client=None, pool_size=8):
+    """
+    Using ODBC  to create ImpalaClient connection.
+
+    Parameters
+    ----------
+    dsn : str, optional
+        Data source name as given in the (unix) odbc.ini file or (Windows) 
+        ODBC Data Source Administrator tool.
+    connection_string : str, optional
+        Preformatted ODBC connection string. Specifying this and dsn at the same
+        time raises ParameterError
+    turbodbc_options : dict, optional
+        to be passed to turbodbc_options
+    
+    Returns
+    -------
+    ImpalaClient
+    """
+
+    params = {'con_string': connection_string,
```

Comment:
Why not just pass these directly instead of putting them in a `dict` and then `**`-ing them in.